ext/intl/converter/converter.c

```diff
@@ -376,25 +376,31 @@ static bool php_converter_set_encoding(php_converter_object *objval,
 			/* Should never happen */
 			actual_encoding = "(unknown)";
 		}
-		php_error_docref(NULL, E_WARNING, "Ambiguous encoding specified, using %s", actual_encoding);
+		char *msg;
+		spprintf(&msg, 0, "Ambigious encoding specified, using %s", actual_encoding);
+		intl_error_set(NULL, error, msg, 1);
```

Comment:
I think this one should remain a warning, as `ucnv_open()` was successful and the conversion is actually performed:

``` php
$str = "\x0a";
var_dump(bin2hex($str));

$str = UConverter::transcode("\x0a", 'UTF-16', 'UTF-8');
var_dump(bin2hex($str));

$str = UConverter::transcode($str, 'windows-1251', 'UTF-16');
var_dump(bin2hex($str));
```

Result:

```
string(2) "0a"
string(8) "fffe0a00"

Warning: UConverter::transcode(): Ambiguous encoding specified, using ibm-5347_P100-1998 in test.php on line 8
string(2) "0a"
```